tests/signal_test.c

```diff
@@ -45,8 +45,7 @@ void* stack_addr = NULL;
 
 void signal_handler(int signal)
 {
-   char buf[128];
-   stack_addr = buf;
+   asm volatile("movq %%rsp, %0" : "=r"(stack_addr));
```

Comment:
Does this fix something or is it just a better way of doing things?
Just wondering.